### ISSUE TITLE
autophagic material in muscle biopsy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7192]
**Preferred term label:**
Uncertain -- see https://pubmed.ncbi.nlm.nih.gov/32403198/ for OMIM 619518 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as Skeletal muscle autophagosome accumulation
